### ISSUE TITLE
How to freeze some heads for finetuning on different datasets?

### ISSUE THREAD

**Issue by yxchng** [open-mmlab/mmdetection#688]
For example, I want to freeze the classification and detection heads and only train the mask head.

**Comment by hellock** [comment]
This is the same as what you do to freeze some pytorch modules.

**Comment by yxchng** [comment]
@hellock I know how to do it in normal PyTorch script. But in mmdetection, there is a lot of encapsulation and I am wondering if there is any easy way to do so?

Do I have to modify `tools/train.py`?

**Comment by hellock** [comment]
Currently there are no easy way to do so and you need to modify the optimizer (and/or the model).

**Comment by leemengwei** [comment]
@yxchng Hi, have you solve the issue? Could you offer some specific instructions?
